templates/install_puppet.ps1.erb

```diff
@@ -257,6 +318,7 @@ try {
   if ($UseLockedFilesWorkaround) {
     Reset-PuppetresDLL $temp_puppetres
   }
+  Reset-PuppetServices $services_before

```

Comment:
May be worth having another try block so that we still reset the services in case something goes wrong with the PuppetresDLL stuff?